src/Raven.Studio/typescript/viewmodels/resources/clusterDashboard.ts

```diff
@@ -240,6 +241,7 @@ class clusterDashboard extends viewModelBase {
                 });
         } else {
             this.addWidget(new cpuUsageWidget(this));
+            //this.addWidget(new ioStatsWidget(this)); - we don't show this by default - as it is only available on linux machines
```

Comment:
better to add it by default to posix

Reply:
do we want to detect at this point if any of cluster item is linux ?